docs/chapters/matrixsplit/content.zh-CN.md

```diff
@@ -0,0 +1,567 @@
+  \cdot
+  \begin{bmatrix}
+  P_1 \\ P_2 \\ P_3 \\ P_4
+  \end{bmatrix}
+\]
+
+假设要在点`t = z`处分割曲线得到两条新的（且显然更小的）贝塞尔曲线，则可以用矩阵表示和一点线性代数求出这两条贝塞尔曲线的坐标。首先将实际的“线上点”信息分离到新的矩阵乘法中：
+
+\[
+  B(t) =
+  \begin{bmatrix}
+  1 & (z \cdot t) & (z \cdot t)^2
+  \end{bmatrix}
+  \cdot
+  \begin{bmatrix}
+   1 &  0 & 0 \\
+  -2 &  2 & 0 \\
+   1 & -2 & 1
+  \end{bmatrix}
+  \cdot
+  \begin{bmatrix}
+  P_1 \\ P_2 \\ P_3
+  \end{bmatrix}
+  =
+  \begin{bmatrix}
+  1 & t & t^2
+  \end{bmatrix}
+  \cdot
+  \begin{bmatrix}
+  1 & 0 & 0 \\
+  0 & z & 0 \\
+  0 & 0 & z^2
+  \end{bmatrix}
+  \cdot
+  \begin{bmatrix}
+   1 &  0 & 0 \\
+  -2 &  2 & 0 \\
+   1 & -2 & 1
+  \end{bmatrix}
+  \cdot
+  \begin{bmatrix}
+  P_1 \\ P_2 \\ P_3
+  \end{bmatrix}
+\]
+
+以及
+
+\[
+  B(t) =
+  \begin{bmatrix}
+  1 & (z \cdot t) & (z \cdot t)^2 & (z \cdot t)^3
+  \end{bmatrix}
+  \cdot
+  \begin{bmatrix}
+   1 &  0 &  0 & 0 \\
+  -3 &  3 &  0 & 0 \\
+   3 & -6 &  3 & 0 \\
+  -1 &  3 & -3 & 1
+  \end{bmatrix}
+  \cdot
+  \begin{bmatrix}
+  P_1 \\ P_2 \\ P_3 \\ P_4
+  \end{bmatrix}
+  =
+  \begin{bmatrix}
+  1 & t & t^2 & t^3
+  \end{bmatrix}
+  \cdot
+  \begin{bmatrix}
+  1 & 0 & 0   & 0\\
+  0 & z & 0   & 0\\
+  0 & 0 & z^2 & 0\\
+  0 & 0 & 0   & z^3
+  \end{bmatrix}
+  \cdot
+  \begin{bmatrix}
+   1 &  0 &  0 & 0 \\
+  -3 &  3 &  0 & 0 \\
+   3 & -6 &  3 & 0 \\
+  -1 &  3 & -3 & 1
+  \end{bmatrix}
+  \cdot
+  \begin{bmatrix}
+  P_1 \\ P_2 \\ P_3 \\ P_4
+  \end{bmatrix}
+\]
+
+如果可以将这些矩阵组合成**[t值] · [贝塞尔矩阵] · [列矩阵]**的形式且前两项保持不变，那么右侧的列矩阵即为描述从`t = 0`到`t = z`的第一段新的贝塞尔曲线的坐标。利用线性代数的一些简单的法则可以很轻松地做到这一点。（如果不在乎推导过程，那么可以直接跳到方框底部得到结果！）
+
+<div class="note">
+
+## 推导新的凸包坐标
+
+推导分割曲线后所得两段曲线的坐标要花上几步，而且曲线次数越高，花的工夫越多，因此先看二次曲线：
+
+\[
+  B(t) =
+  \begin{bmatrix}
+  1 & t & t^2
+  \end{bmatrix}
+  \cdot
+  \begin{bmatrix}
+  1 & 0 & 0 \\
+  0 & z & 0 \\
+  0 & 0 & z^2
+  \end{bmatrix}
+  \cdot
+  \begin{bmatrix}
+   1 &  0 & 0 \\
+  -2 &  2 & 0 \\
+   1 & -2 & 1
+  \end{bmatrix}
+  \cdot
+  \begin{bmatrix}
+  P_1 \\ P_2 \\ P_3
+  \end{bmatrix}
+\]
+
+\[
+  =
+  \begin{bmatrix}
+  1 & t & t^2
+  \end{bmatrix}
+  \cdot
+  \underbrace{\kern 2.25em Z \cdot M \kern 2.25em}_{\textit{将这一项……}}
+  \cdot
+  \begin{bmatrix}
+  P_1 \\ P_2 \\ P_3
+  \end{bmatrix}
+\]
+
+\[
+  =
+  \begin{bmatrix}
+  1 & t & t^2
+  \end{bmatrix}
+  \cdot
+  \underbrace{ M \cdot M^{-1} \cdot Z \cdot M }_{\textit{变为这一项后……}}
+  \cdot
+  \begin{bmatrix}
+  P_1 \\ P_2 \\ P_3
+  \end{bmatrix}
+\]
+
+\[
+  =
+  \begin{bmatrix}
+  1 & t & t^2
+  \end{bmatrix}
+  \cdot
+  M
+  \underbrace{ \kern 1.25em \cdot \kern 1.25em Q \kern 1.25em \cdot \kern 1.25em}_{\textit{……得到这一项！}}
+  \begin{bmatrix}
+  P_1 \\ P_2 \\ P_3
+  \end{bmatrix}
+\]
+
+以上变形可行是因为[*M · M<sup>-1</sup>*]为单位矩阵。这有点像在计算中让某项乘以x/x——不改变函数本身，却可以将函数改写为更好处理的形式或者得到不同的分解。类似地将上式中的矩阵左乘以[*M · M<sup>-1</sup>*]不会影响整个式子，却可以将矩阵序列[*something · M*]变为[*M · something*]，而这至关重要——如果知道了[*M<sup>-1</sup> · Z · M*]是什么，那就可以将其施加到已有坐标上得到一条二次贝塞尔曲线的标准矩阵表示（即[*T · M · P*]）)和表示从*t = 0*到*t = z*的曲线的一系列新坐标。计算如下：
```

Comment:
it probably makes sense to translate "something" to the Chinese equivalent here, too.